lib/plugins/autolinks.js

```diff
@@ -13,23 +13,33 @@ module.exports = class Autolinks extends Diffable {
   }
 
   comparator (existing, attr) {
-    return existing.key_prefix === attr.key_prefix && existing.url_template === attr.url_template
+    return existing.key_prefix === attr.key_prefix &&
+      existing.url_template === attr.url_template
```

Comment:
I realise you haven't changed this but it seems that since `key_prefix` must be unique that this comparison is more correct
```suggestion
    return existing.key_prefix === attr.key_prefix
```